src/macos.rs

```diff
@@ -245,6 +253,18 @@ pub(crate) fn configure_dns_servers(dns_servers: &[IpAddr]) -> std::io::Result<(
     Ok(())
 }
 
+fn config_dns_servers_empty(original_gw_scope: &str) -> std::io::Result<()> {
+    let script = format!(
+        r#"
+    original_gw_scope={}
+    networksetup -setdnsservers "$original_gw_scope" empty
+    "#,
+        original_gw_scope
+    );
+    let _r = run_command("sh", &["-c", &script])?;
```

Comment:
Does the indirection through the shell provide us with any benefits here? Calling `networksetup` directly is probably safer, since the assignment to `original_gw_scope` might potentially allow shell command injection.